docs/main/01-quick-start.md

```diff
@@ -4,6 +4,20 @@ title: Quick Start
 sidebar_label: Quick Start
 ---
 
+:::info
```

Comment:
IMO, this info section should come after the title ("_Quick Start_") similarly to the "Installing Kratix" sections.